### PR TITLE
Created Footer

### LINKED ISSUE #17: Make a footer component.
Kindly refer to the design and make the footer. All the dimensions should be pixel perfect. Use scss variables wherever necessary.

### PR DESCRIPTION
**Pull request by joharikushagra**
Fix #17 
created footer component

### CODE REVIEW

**Review by salil-naik — CHANGES_REQUESTED** [review]
Foundation grid is used perfectly. 

- [ ] Add footer screenshot
- [ ] Remove `ul` from `p`. `p` is not required there. 
- [ ] SCSS filename should be footer.scss. Remove underscore. 
- [ ] Give the following classes

> `footer__sub-title` (remove text__head)
> `footer__nav-item-container` (for ul)
> `footer__nav-item` (for li)
> replace vote__of__thanks with `footer__credits`

**Comment by joharikushagra** [comment]
okay I'll make the changes

**Comment by joharikushagra** [comment]
footer1

File: frontend/src/css/components/footer.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+@import '../settings//global';
+
+$footer-bg : #e7edf3;
+
+.footer{
+    background-color:$footer-bg;
+    padding: 0 101px;
+    .grid-container{
```
Comment (salil-naik) [inline frontend/src/css/components/footer.scss:8]: never overwrite frameworks classes like grid-x, grid-container, cell, etc.

File: frontend/src/css/components/footer.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        padding: 54px 0;
+    }
+    .cell{
```
Comment (salil-naik) [inline frontend/src/css/components/footer.scss:11]: framework class

File: frontend/src/css/components/footer.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+.footer__nav-item{
+    text-align: left;
+    list-style: none;  
+    line-height: 40px;  
+    @extend .text;
+}
+
+.large-2{
```
Comment (salil-naik) [inline frontend/src/css/components/footer.scss:27]: framework class

File: frontend/src/css/components/footer.scss [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    .cell{
+        padding:10px 10px;
+    }
+    .footer__sub-title{
+        color: $title-color;
+    }
+}
+
+
+.footer__nav-item{
```
Comment (salil-naik) [inline frontend/src/css/components/footer.scss:20]: You could have used scss nesting.

Comment (joharikushagra) [reply frontend/src/css/components/footer.scss:20]: Okay I'll remember that:)

**Pull request merged**